.test-infra/metrics/build.gradle

```diff
@@ -27,7 +27,7 @@ ext {
   cluster = 'metrics'
   zone='us-central1-a'
   projectName='apache-beam-testing'
-  kubeConfigPath='/home/jenkins/.kube/config'
+  kubeConfigPath = project.properties['KUBE_CONFIG_PATH'] ?: '/home/jenkins/.kube/config'
```

Comment:
(just a note) may clean up this once completely migrate off jenkins (`'/home/jenkins/.kube/config'` -> `"/home/${System.getenv('ENVIRONMENTAL')}/.kube/config"`)